pyOCD/tools/gdb_server.py

```diff
@@ -65,6 +65,7 @@ def build_parser(self):
         parser.add_argument("-t", "--target", dest="target_override", choices=supported_targets, default=None, help="Override target to debug.  Supported targets are: " + ", ".join(supported_targets), metavar="TARGET")
         parser.add_argument("-n", "--nobreak", dest="break_at_hardfault", default=True, action="store_false", help="Disable halt at hardfault handler.")
         parser.add_argument("-r", "--reset-break", dest="break_on_reset", default=False, action="store_true", help="Halt the target when reset.")
+        parser.add_argument("-C", "--vector-catch", default='', help="Select enabled vector catch options, one letter per enabled source in any order. (h=hard fault, b=bus fault, m=mem fault, i=irq err, s=state err, c=check err, p=nocp, r=reset, a=all, n=none)")
```

Comment:
(It would be 'n' rather than "none".) It could, but it doesn't really matter. It only checks for the characters to enable catch options; it doesn't actually check for the none option. But I thought it would be good to document a char you could use for none, since that's easier to type on a command line than an empty string (i.e., `-Cn` versus `-C ""`).